src/styles/main.scss

```diff
@@ -1,3 +1,66 @@
-body {
-  margin: 0;
+$blockSize: 100px;
+$gapSize: 1px;
+$weekDays:
+  mon 0,
+  tue 1,
+  wed 2,
+  thu 3,
+  fri 4,
+  sat 5,
+  sun 6;
+
+.calendar {
+  position: absolute;
+  display: flex;
+  flex-wrap: wrap;
+
+  padding: $blockSize;
+  width: (7 * $blockSize) + ($gapSize * 6);
+  gap: $gapSize;
+
+  &--month-length {
+    @for $day from 28 through 30 {
+      &--#{$day} .calendar__cell:nth-child(n+#{$day + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &--start-day {
+    @each $day, $position in $weekDays {
+      &--#{$day} .calendar__cell:first-child {
+        margin-left: $position * ($blockSize + $gapSize);
+      }
+    }
+  }
+
+  &__cell {
+    display: block;
+    box-sizing: border-box;
+    text-align: center;
+
+    width: $blockSize;
+    height: $blockSize;
+    line-height: $blockSize;
+
+    background-color: #eee;
+    border: 1px solid #000;
+    transition: transform 0.5s;
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i}) {
+        &::before {
+          content: "#{$i}";
+          font-family: Arial, Helvetica, sans-serif;
+          font-size: 30px;
```

Comment:
Leave only content here. You can add all other properties just with `calendar__cell` selector, no need to use loop for that
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/0c1481db-cc99-4901-a374-18c0881c3040)

Reply:
Can we use formula to change font-size depending on cell size,
  ignoring the README.md file ? If we are prepared for cell size change, it would be great
  to have a formula for font